api/client/swarm/opts.go

```diff
@@ -30,9 +30,11 @@ const (
 
 var (
 	defaultPolicies = []swarm.Policy{
-		{Role: WORKER, Autoaccept: true},
-		{Role: MANAGER, Autoaccept: false},
+		{Role: worker, Autoaccept: true},
+		{Role: manager, Autoaccept: false},
 	}
+
+	exists = struct{}{}
```

Comment:
I wonder what the point of using `struct{}` instead of bool. `bool` syntax is much simpler.